basyx/aas/adapter/http.py

```diff
@@ -875,6 +884,48 @@ def delete_submodel_submodel_elements_id_short_path(self, request: Request, url_
         self._namespace_submodel_element_op(parent, parent.remove_referable, id_short_path[-1])
         return response_t()
 
+    def get_submodel_submodel_element_attachment(self, request: Request, url_args: Dict, **_kwargs) \
+            -> Response:
+        submodel_element = self._get_submodel_submodel_elements_id_short_path(url_args)
+        if isinstance(submodel_element, model.Blob):
+            return Response(submodel_element.value, content_type=submodel_element.content_type)
+        elif isinstance(submodel_element, model.File):
+            try:
+                with open(submodel_element.value, 'rb') as file:
+                    file_content = file.read()
+                return Response(file_content, content_type=submodel_element.content_type)
+            except FileNotFoundError:
+                return Response("File not found", status=404)
+            except Exception as e:
+                return Response("Internal server error", status=500)
+        return Response("Unsupported submodel element type", status=400)
+
+    def put_submodel_submodel_element_attachment(self, request: Request, url_args: Dict, **_kwargs) -> Response:
+        submodel_element = self._get_submodel_submodel_elements_id_short_path(url_args)
+        if 'file' in request.files:
+            file_storage: FileStorage = request.files['file']
+            if isinstance(submodel_element, model.Blob):
+                submodel_element.value = file_storage.read()
+            submodel_element.commit()
+            return Response("File uploaded successfully", status=200)
```

Comment:
This should also return the appropriate response type for a request, i.e. `get_response_type()`. Furthermore, this should be `204` instead of `200` (when using the response type returned by `get_response_type()`, simply returning an empty response automatically sets this to `204`).